packages/stratocacher-layer-dynamo/src/index.js

```diff
@@ -0,0 +1,97 @@
+import Q from "q";
+import {DynamoDB} from "aws-sdk";
+import {Layer} from "stratocacher";
+
+var CLIENT;
+
+class Client {
+	constructor(awsConfig) {
+		const ddbClient = new DynamoDB(awsConfig);
+		this.getItem = Q.nbind(ddbClient.getItem, ddbClient);
+		this.putItem = Q.nbind(ddbClient.putItem, ddbClient);
+	}
+}
+
+/*
+	DynamoDB Caching Layer.
+	Uses AWS-SDK Dynamo client: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html
+	Configuration requires a table name corresponding to Dynamo configuration.
+	Your table must look like this:
+
+	field:  type:
+	key     String
+	v       String
+	t       Number
+	i       Boolean
+*/
+export default class LayerDynamo extends Layer {
+
+	constructor(options) {
+		super(options);
+		if (!this.opt.tableName) {
+			throw new Error("Must provide tableName");
```

Comment:
What would the behavior be if these errors were thrown from the `Client` constructor?

Reply:
In that case the error would be thrown when `#get` is called.

Reply:
And would it be captured and emitted as an `error` event?

Reply:
I don't believe it would. I added a commit to emit error events rather than throwing them.